Mklinks subprocess -- rcsdiff -c -rR4-Vanilla -rR4-Mvex-r1 extensions/server/RCS/Imakefile,v 
*** 1.1.1.1	1991/09/26 20:18:16
--- 1.2.1.1	1991/09/23 00:37:23
***************
*** 1,3 ****
--- 1,4 ----
+ # $Header: /usr/local/src/video/release/mvex-r1/extensions/server/RCS/Imakefile,v 1.2.1.1 1991/09/23 00:37:23 toddb Exp $
  #include <Server.tmpl>
  IMAKE_DEFINES = $(EXT_DEFINES)
  
***************
*** 40,42 ****
--- 41,48 ----
  #endif
  
  DependTarget()
+ 
+ #if BuildMvex
+ all::
+ 	cd ../mvex/server; $(MAKE) $(MFLAGS) 'CDEBUGFLAGS=$(CDEBUGFLAGS)'
+ #endif
